frontend/src/components/layouts/components/Content.tsx

```diff
@@ -0,0 +1,15 @@
+import { Layout } from 'antd';
+import styled from 'styled-components';
+
+import { Colors } from '../../../styles';
+
+const { Content: AContent } = Layout;
+
+const Content = styled(AContent)`
+  background-color: ${Colors.onPrimary};
+  box-shadow: 0px 5px 30px -6px rgba(100, 100, 100, 0.5);
+  border-radius: 10px;
+  margin: 25px 20px;
```

Comment:
Margines w sumie też, zamiast tego lepiej jest dać `padding`. Margines spowoduje to, że może dookoła może pojawić się inny kolor (np. z body). Ten komponent powinien wypełniać całą wolną przestrzeń.